src/Tests/dotnet-new.Tests/Approvals/DotnetVisualBasicClassTemplatesTest.module.langVersion=9.targetFramework=netstandard2.0.verified/ClassLib.vbproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>ClassLib</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>9</LangVersion>
```

Comment:
Please note, that the language version for VB are totally different from C#:
https://learn.microsoft.com/en-us/dotnet/visual-basic/language-reference/configure-language-version

I am pretty sure, that we don't want a language version here which equals to the VS 2008 compiler (if I remember the versions correctly.)